### ISSUE TITLE
issues with installing Pardiso

### ISSUE THREAD

**Issue by manschuer** [KristofferC/Pardiso.jl#76]
Hi all,

I've tried to install Pardiso in Julia on my Linux notebook.

I've downloaded libpardiso600-GNUXXX-X86-64.so to "/Users/manuel/Pardiso" and set
ENV["JULIA_PARDISO"] = "/Users/manuel/Pardiso" in .julia/config/startup.jl.

I can run 

Pkg.add("Pardiso")
Pkg.build("Pardiso")
using Pardiso

without any error message, but 

Pardiso.show_build_log()
gives 
"no log file found"

and 
PardisoSolver()
gives 
"pardiso library was not loaded"

Which step did I miss?
Thank you very much for your help!

**Comment by j-fu** [comment]
How did you handle the license ?

**Comment by manschuer** [comment]
hmm, good question.. after I've registered I got a key, but I never used it..
how should I handle this?

thanks you very much..

**Comment by j-fu** [comment]
Hi this is from the pardiso manual:

3  Linking and Running the Software

A general remark for Unix operating systems is that the environment variableLD_LIBRARY_PATHmust bepointing to the PARDISO library. The environment variableOMP_NUM_THREADSmust be set in all casesof using the multi-threaded library version.The software can be downloaded from the PARDISO page athttp://www.pardiso-project.org. Alicense filepardiso.licis required for the execution of the solver and it will be generated at the downloadarea of the web page. The user must supply the user identification (Unix: “whoami”, Windows: usernameexactly as Windows has it). This license file must always be present either in the home directory ofthe user or in the directory from which the user runs a program linked with the PARDISO library.Alternatively, this file can be placed in a fixed location and the environment variablePARDISO_LIC_PATHmust be set to the path of its location

**Comment by j-fu** [comment]
Are you sure you have Pardiso 6 and the license for that version ?

**Comment by manschuer** [comment]
> Are you sure you have Pardiso 6 and the license for that version ?

Yes, I have libpardiso600-GNU720-X86-64.so and for the license "PARDISO 6.0 Academic License".

I put now also the file 'pardiso.lic' with the key inside in my home directory.

Still not loading, but no error message.

What could it be else? Or what could I test?

**Comment by j-fu** [comment]
In fact  am wondering how you have got the pardiso lib and the license for 6.0. IMHO these are from 2018, and it is not possible anymore to download them. This means that your license may have expired (as Pardiso academic licenses last one year).

**Comment by manschuer** [comment]
Hmm, I just requested the license from the official page which results in license 6.0. The link mentioned in the mail for downloading refers me to package 6.0 as well. How can I get the current?

**Comment by j-fu** [comment]
Ok it seems that they silently dialed back the downloadable version and the  license to version 6.
I checked this on my installation, and it works, appearantly with the same file you have.

You may try to repeat the  installation after completely removing  ~/.julia/packages/Pardiso after a failed
attempt.

**Comment by manschuer** [comment]
Ok thank you. Unfortunately in my case it didn't work. Any other ideas what I could do?

**Comment by j-fu** [comment]
What is your OS + gcc version ? 
AFAIK there is also libpardiso600-GNU800-X86-64.so  available. If your OS is not too old , this may be the better try.

Otherwise please retry all steps and post them here.
